language-design.md

```diff
@@ -73,8 +101,10 @@ sympathize more with the tenor of the article or the tenor of the majority of
 the commenters?
 
 **Response**
+The essence of this conflict I think is logical vs non-logical syntax. Personally, I find some solace in programming because it is completely logical. The English language is not completely logical but I still enjoy using it for exactly that reason. English is a language all the same as any programming language. We could imagine that eventually, English will be a programming language and we will be able to speak it to computers and have them understand it. But technology is not that far along yet so as of now we have to speak to computers in a more logical language. This is solely because computers think with electrical circuits and bit (purely logical) and humans think with brain cells (which leaves room for interpretation). Some people would prefer computers think more like humans, but other technical minded people may always prefer to think like computers. -Daniel
 
-
+I'm somewhat inclined to agree with the less arrogant and condescending commenters, for example the commenter who suggested that the tools of programming are no more complicated than those of any other technical profession. But there does seem to be a sense of elitism among many of them that I really can't get behind, and I can only agree with the above sentiment so far as general-purpose programming languages are concerned. I think that telling a computer what to do is something that is
+useful for everyone to be able to do, and (I can't say with full confidence since I've only been in this class for a couple of weeks, but...) I think that's where DSLs come in. So I guess I'd say I agree with the commenters that things like Quorum shouldn't be general-purpose programming languages, but I would say that good DSLs might look a lot like Quorum does. -Aaron
 
```

Comment:
I think what a lot of the more arrogant and condescending commenters were going for was drawing the line between programming and programming _well_. I can see how it can very easily come off as arrogant (some of the commenters were simply super rude). I think the idea that a lot of the commenters were going for was that syntax is just the tip of the iceberg, so what's the big deal? Not that I necessarily agree with the sentiment, but it's an interesting point of discussion whether or not the tools of programming are truly no more complicated than those of any other technical profession.